js/ui/camera.js

```diff
@@ -303,32 +303,58 @@ class Camera extends Evented {
      *     {@link Map#easeTo}. If `false`, the map transitions using {@link Map#flyTo}. See
      *     {@link Map#flyTo} for information about the options specific to that animated transition.
      * @param {Function} [options.easing] An easing function for the animated transition.
-     * @param {number} [options.padding=0] The amount of padding, in pixels, to allow around the specified bounds.
+     * @param {number|Object} [options.padding] Sets padding around the given bounds on each side in pixels. Accepts a number for all padding edges or an object with padding for each compass direction (north, south, east, and west)
```

Comment:
I just did a little experimentation with this PR and discovered that `Map#fitBounds` always resets the bearing to 0 when it is called. 🤔 

We may be able to rename these properties, ship this PR as-is, and open a separate issue about preserving bearing through a `Map#fitBounds`.